src/topology/algebra/ring.lean

```diff
@@ -23,7 +23,8 @@ of topological (semi)rings.
 
 - `subring.topological_closure`/`subsemiring.topological_closure`: the topological closure of a
   `subring`/`subsemiring` is itself a `sub(semi)ring`.
-- `prod_ring`/`prod_semiring`: The product of two topological (semi)rings.
+- The product of two topological (semi)rings.
+- The arbitrary product of topological (semi)rings.
```

Comment:
```suggestion
- `pi.topological_semiring`/`pi.topological_ring`: The arbitrary product of
  topological (semi)rings.
```

Reply:
I edited it as:
- `prod.topological_semiring`/`prod.topological_ring`: The product of two topological (semi)rings.
- `pi.topological_semiring`/`pi.topological_ring`: The arbitrary product of topological (semi)rings.